ios/junit4-examples/pom.xml

```diff
@@ -56,26 +56,31 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.12.4</version>
+                <version>2.22.1</version>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
please specify the version for the same.
https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-compiler-plugin